### PR TITLE
python3Packages.angr: 9.0.6885 -> 9.0.7491

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 9.0.7491

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 123838` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>16 packages built:</summary>
  <ul>
    <li>python38Packages.ailment</li>
    <li>python38Packages.angr</li>
    <li>python38Packages.angrop</li>
    <li>python38Packages.archinfo</li>
    <li>python38Packages.claripy</li>
    <li>python38Packages.cle</li>
    <li>python38Packages.pysmt</li>
    <li>python38Packages.pyvex</li>
    <li>python39Packages.ailment</li>
    <li>python39Packages.angr</li>
    <li>python39Packages.angrop</li>
    <li>python39Packages.archinfo</li>
    <li>python39Packages.claripy</li>
    <li>python39Packages.cle</li>
    <li>python39Packages.pysmt</li>
    <li>python39Packages.pyvex</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 123838` at 11788b82 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>10 packages marked as broken and skipped:</summary>
  <ul>
    <li>python38Packages.ailment</li>
    <li>python38Packages.angr</li>
    <li>python38Packages.angrop</li>
    <li>python38Packages.cle</li>
    <li>python38Packages.pyvex</li>
    <li>python39Packages.ailment</li>
    <li>python39Packages.angr</li>
    <li>python39Packages.angrop</li>
    <li>python39Packages.cle</li>
    <li>python39Packages.pyvex</li>
  </ul>
</details>
<details>
  <summary>6 packages built successfully:</summary>
  <ul>
    <li>python38Packages.archinfo</li>
    <li>python38Packages.claripy</li>
    <li>python38Packages.pysmt</li>
    <li>python39Packages.archinfo</li>
    <li>python39Packages.claripy</li>
    <li>python39Packages.pysmt</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `wheel`.
Near pkgs/development/python-modules/pysmt/default.nix:4:3:
```
  |
4 | , wheel
  |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md">python-include-tests</a>

Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/ailment/default.nix:27:0:
```
   |
27 |     description = "The angr Intermediate Language";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/pysmt/default.nix [hunk 1]
```diff
@@ -1,28 +1,21 @@
...
-, pytestCheckHook
-, six
+, fetchPypi
 }:
 
 buildPythonPackage rec {
   pname = "pysmt";
-  version = "0.9.0";
+  version = "0.9.1.dev132";
+  format = "wheel";
```
Comment (jonringer) [inline pkgs/development/python-modules/pysmt/default.nix:9]: why the wheel?

Comment (fabaff) [reply pkgs/development/python-modules/pysmt/default.nix:9]: Upstream is releasing the `.devX` versions only as wheels on PyPI. `dev` releases on GitHub are not tagged.

Comment (jonringer) [reply pkgs/development/python-modules/pysmt/default.nix:9]: -_- sigh.

Can you add a comment?
```suggestion
  format = "wheel"; # dev versions are only distributed as wheels
```

Comment (fabaff) [reply pkgs/development/python-modules/pysmt/default.nix:9]: Point taken

Request for tags: https://github.com/pysmt/pysmt/issues/693

**Pull request merged**